extensions/adobe/experience/adcloud/search/admeasuresearch-adcloud.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/search/admeasuresearch-adcloud",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Ad Cloud Aggregated Measures Time Series Class",
```

Comment:
@cfraser - Since this class will appear int he UI along side the existing "XDM ExperienceEvent" and "XDM Profile". What would you like them to name the class here?

Reply:
Hi @cmathis @cfraser 
I updated the title and description to reflect the purpose and usage restriction of this ad-cloud solution specific class. Let me know if new title and description makes sense.